module-services/service-evtmgr/service-evtmgr/WorkerEventCommon.hpp

```diff
@@ -49,15 +49,14 @@ class WorkerEventCommon : public sys::Worker
     sys::Service *service = nullptr;
 
   private:
-    static constexpr auto stackDepthBytes = 3072;
+    static constexpr auto stackDepthBytes = 1024 * 4;
```

Comment:
is stack increase needed?

Reply:
Analysis with `puregdb` shown that EventWorker sometimes has only ~17% of free stack, I increased it just in case.

![image](https://user-images.githubusercontent.com/26345399/186671412-80ef58a9-9ef6-495c-96ff-7973ce595df8.png)